atos/modules/MQTTBridge/inc/mqttsubscribercallback.hpp

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include <mqtt/async_client.h>
+#include <mqtt/callback.h>
+
+
+class MQTTSubscriberCallback : public virtual mqtt::callback, public virtual mqtt::iaction_listener {
+
+	public:
+		MQTTSubscriberCallback(mqtt::async_client& client, mqtt::connect_options& connectionOptions, std::vector<std::string>& topics);
+		~MQTTSubscriberCallback();
+
+		std::string mqttMessage = "";
+
+	private:
+		mqtt::async_client& client;
+		mqtt::connect_options& connectionOptions;
+		std::vector<std::string> topics;
```

Comment:
Why is topics also not a pointer? Am I missing something?

Reply:
Subscriber callbacks are created per topics that are subscribed. So i don't think i makes much sense to have it as a pointer.